crates/shared/src/price_estimation.rs

```diff
@@ -519,6 +520,27 @@ pub fn amounts_to_price(sell_amount: U256, buy_amount: U256) -> Option<BigRation
     ))
 }
 
+#[derive(prometheus_metric_storage::MetricStorage)]
+pub(in crate::price_estimation) struct Metrics {
```

Comment:
Same for `inc_estimator()`.
```suggestion
struct Metrics {
```